app.js

```diff
@@ -31,9 +31,36 @@
 // 74 - Soft Tek Front
 
 let drums = null;
+let buttons = null;
+const config = {}
+const DEFAULT_SOUNDFONT =  'Doumbek-Faisal';
+const DEFAULT_INSTRUMENT = 'doumbek';
+const DEFAULT_BUTTONS = {
+  "b1": 64,
+  "b2": 65,
+  "b3": 66,
+  "b4": 67,
+  "b5": 68,
+  "b6": 69
+};
+
+async function loadSoundFonts(soundfont) {
+  const soundFonts = await fetch('/soundfonts.json')
+    .then(data => data.json())
+    .then(json => json)
+  return soundFonts[soundfont]
+}
+
+async function loadConfig(soundfont, instrument) {
+  if (!config[soundfont]) {
+    config[soundfont] = await loadSoundFonts(soundfont);
+  }
+  return config[soundfont][instrument];
+}
+
 async function loadInstrument() {
   return new Promise((resolve, reject) => {
-    Soundfont.instrument(new AudioContext(), 'doumbek', { soundfont: 'Doumbek-Faisal', nameToUrl: function(name, soundfont, format) {
+    Soundfont.instrument(new AudioContext(), DEFAULT_INSTRUMENT, { soundfont: DEFAULT_SOUNDFONT, nameToUrl: function(name, soundfont, format) {
```

Comment:
It does not make sense to use the defaults in this function. The soundfont and instrument should be passed as arguments to the function.